grommet/grommet
# Normalize theme.checkBox.toggle.color

#### What does this PR do?

This PR fixes #5271 by removing the light/dark switch and relying on normalizeColor to normalize the color spec for light/dark instead.

#### Where should the reviewer start?

#### What testing has been done on this PR?

Manual testing by adjusting the CheckBox story to use color strings.

#### How should this be manually tested?

1. Run the storybook
2. Check the custom toggle story: http://localhost:9001/?path=/story/input-checkbox-custom-toggle--custom-toggle
3. Update src/js/components/CheckBox/stories/CustomToggle.js to use color strings instead of objects
4. Confirm results are the same

#### Any background context you want to provide?

#### What are the relevant issues?

#5271 

#### Screenshots (if appropriate)

#### Do the grommet docs need to be updated?

Yes... the docs currently say the following:

![image](https://user-images.githubusercontent.com/1895462/118897414-34032480-b8d8-11eb-8bce-553264eb7f18.png)

But I believe it should instead say `string | { 'dark': string, 'light': string }`

It looks like it is displaying the `defaultValue` instead of the `type` from the docs here: https://github.com/grommet/grommet/blob/master/src/js/components/CheckBox/doc.js#L158-L162

I wasn't sure how to go about updating it to show the type value instead though.

#### Should this PR be mentioned in the release notes?

#### Is this change backwards compatible or is it a breaking change?

This is backward compatible
